src/envo/envo-edit.owl

```diff
@@ -7241,14 +7237,12 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000148> ObjectSomeValuesFrom(<
 
 AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "MA:ma"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000149> "Chains of rocks or coral at or near the surface of water constructed by man."^^xsd:string)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_00000149> "https://en.wikipedia.org/wiki/Artificial_reef"^^xsd:string)
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> <http://purl.obolibrary.org/obo/ENVO_00000149> "ENVO"^^xsd:string)
-AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/ENVO_00000149> "ENVO:00000149"^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_00000149> "artificial reef"^^xsd:string)
-SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000149> <http://purl.obolibrary.org/obo/ENVO_00000130>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000149> <http://purl.obolibrary.org/obo/ENVO_01001899>)
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000150> (coral reef)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Coral_reef"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000150> "Aragonite structures produced by living organisms, found in shallow, marine waters with little nutrients in the water."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Coral_reef"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000150> "A marine reef which is composed primarily of aragonite structures produced by living organisms, found in shallow, low-nutrient waters."@en)
```

Comment:
I'd suggest 'calcium-rich carbonate minerals'.   If its a 'coral' reef, don't the 'living organisms' have to be members of that genus?  Is there such a thing as a 'non-marine' reef??